Gemfile.lock

```diff
@@ -338,4 +342,4 @@ DEPENDENCIES
   xcpretty-travis-formatter
 
 BUNDLED WITH
-   2.2.19
+   2.2.21
```

Comment:
These `Gemfile.lock` are from the update to 1.3.0 to 1.3.1 of the release toolkit.